react-relay/react-relay-tests.tsx

```diff
@@ -45,12 +45,17 @@ export default class AddTweetMutation extends Relay.Mutation<Props, Response> {
 interface ArtworkProps {
     artwork: {
         title: string
-    }
+    },
+    relay: Relay.RelayProp,
```

Comment:
The problem here is that now, anywhere `<Artwork ...>` is used, it will require the relay prop to be passed in. The obvious and easy answer is to make `relay` optional here (e.g. `relay?: Relay.RelayProp` which is probably fine but not 100% accurate.